config/playground/helm/dev/helm-opensearch.yaml

```diff
@@ -0,0 +1,1043 @@
+                - "indices:data/read/plugins/replication/changes"
+                - "indices:data/read/plugins/replication/file_chunk"
+        
+        # Allows users to use all cross cluster replication functionality at follower cluster
+        cross_cluster_replication_follower_full_access:
+          reserved: true
+          cluster_permissions:
+            - "cluster:admin/plugins/replication/autofollow/update"
+          index_permissions:
+            - index_patterns:
+                - '*'
+              allowed_actions:
+                - "indices:admin/plugins/replication/index/setup/validate"
+                - "indices:data/write/plugins/replication/changes"
+                - "indices:admin/plugins/replication/index/start"
+                - "indices:admin/plugins/replication/index/pause"
+                - "indices:admin/plugins/replication/index/resume"
+                - "indices:admin/plugins/replication/index/stop"
+                - "indices:admin/plugins/replication/index/update"
+                - "indices:admin/plugins/replication/index/status_check"
+        
+        # Allow users to read ML stats/models/tasks
+        ml_read_access:
+          reserved: true
+          cluster_permissions:
+            - 'cluster:admin/opensearch/ml/stats/nodes'
+            - 'cluster:admin/opensearch/ml/models/get'
+            - 'cluster:admin/opensearch/ml/models/search'
+            - 'cluster:admin/opensearch/ml/tasks/get'
+            - 'cluster:admin/opensearch/ml/tasks/search'
+        
+        # Allows users to use all ML functionality
+        ml_full_access:
+          reserved: true
+          cluster_permissions:
+            - 'cluster_monitor'
+            - 'cluster:admin/opensearch/ml/*'
+          index_permissions:
+            - index_patterns:
+                - '*'
+              allowed_actions:
+                - 'indices_monitor'
+        
+        # Allows users to use all Notifications functionality
+        notifications_full_access:
+          reserved: true
+          cluster_permissions:
+            - 'cluster:admin/opensearch/notifications/*'
+        
+        # Allows users to read Notifications config/channels
+        notifications_read_access:
+          reserved: true
+          cluster_permissions:
+            - 'cluster:admin/opensearch/notifications/configs/get'
+            - 'cluster:admin/opensearch/notifications/features'
+            - 'cluster:admin/opensearch/notifications/channels/get'
+      roles_mapping.yml: |-
+        ---
+        # In this file users, backendroles and hosts can be mapped to Security roles.
+        # Permissions for OpenSearch roles are configured in roles.yml
+        
+        _meta:
+          type: "rolesmapping"
+          config_version: 2
+        
+        # Define your roles mapping here
+
+        opendistro_security_anonymous:
+          backend_roles:
+          - "opendistro_security_anonymous_backendrole"
```

Comment:
consider create role mappings using pre-defined read access roles, instead of define everything in `opendistro_security_anonymous` role

Reply:
@zengyan-amazon I have tried using role mappings to map read access roles to the `opendistro_security_anonymous_backendrole` but it does not seem to work properly. I was, however, able to leverage the individual permissions of each read access role and that allows the anonymous user to function properly.

Reply:
I believe it can be completed by using roles mappings

Reply:
Done, `opendistro_security_anonymous_backendrole` is now mapped to a `readall` role and several permissions I added in `opendistro_security_anonymous`